### PR TITLE
Added additional configuration options for numberSlider per issue #1720

### PR DESCRIPTION
**Pull request by alexdaversa**
PR to resolve issue #1720

### CODE REVIEW

File: pyrevitlib/pyrevit/forms/__init__.py [hunk 1]
```diff
@@ -3270,7 +3271,7 @@ def ask_for_date(default=None, prompt=None, title=None, **kwargs):
         )
 
 
-def ask_for_number_slider(default=None, min=0, max=100, prompt=None, title=None, **kwargs):
+def ask_for_number_slider(default=None, min=0, max=100, interval=1 prompt=None, title=None, **kwargs):
```
Comment (thumDer) [inline pyrevitlib/pyrevit/forms/__init__.py]: a comma is missing before `prompt`

File: pyrevitlib/pyrevit/forms/__init__.py [hunk 2]
```diff
@@ -3270,7 +3271,7 @@ def ask_for_date(default=None, prompt=None, title=None, **kwargs):
...
     """Ask user to select a number value.
 
     This is a shortcut function that configures :obj:`GetValueWindow` for
```
Comment (thumDer) [inline pyrevitlib/pyrevit/forms/__init__.py:3277]: Please extend the docstring as well

**Review by thumDer — CHANGES_REQUESTED** [review]

File: pyrevitlib/pyrevit/forms/__init__.py (hunk 1 above)
Comment (alexdaversa) [reply pyrevitlib/pyrevit/forms/__init__.py]: @thumDer thanks! will update PR

File: pyrevitlib/pyrevit/forms/__init__.py (hunk 2 above)
Comment (alexdaversa) [reply pyrevitlib/pyrevit/forms/__init__.py:3277]: @thumDer thanks! I extended docstring as well.

**Comment by jmcouffin** [comment]
Well done, @alexdaversa thanks

**Pull request merged**